cmd/kubeapps-apis/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# syntax = docker/dockerfile:experimental
+
+FROM bitnami/golang:1.16.3 as builder
+WORKDIR /go/src/github.com/kubeapps/kubeapps
+COPY go.mod go.sum ./
+COPY pkg pkg
+COPY cmd cmd
+# With the trick below, Go's build cache is kept between builds.
+# https://github.com/golang/go/issues/27719#issuecomment-514747274
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    go build ./cmd/kubeapps-apis
+# Build the current standard plugins
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    go build \
+    -o /kapp-controller-packages-v1alpha1-plugin.so -buildmode=plugin \
+    ./cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/main.go
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    go build \
+    -o /helm-operator-packages-v1alpha1-plugin.so -buildmode=plugin \
+    ./cmd/kubeapps-apis/plugins/helm_operator/packages/v1alpha1/main.go
+
+# Note: unlike the other docker images for go, we cannot use scratch as the plugins
+# are loaded using the dynamic linker.
```

Comment:
Uff, let's see how we will handle it during the official Bitnami image generation

Reply:
Why is that an issue? I don't remember, but is that what caused the difficulties with the pinniped-proxy image? It *shouldn't* be an issue to use a standard base image rather than scratch - keen to understand (maybe not here).

Reply:
Thinking twice... they tend to use `scratch` images but the problem with the pinniped-proxy was caused to the openssl libs (we would have to maintain ourselves these deps, and we don't want to be in charge of updating these deps manually). In this case, as we manage our own deps, it shouldn't be really an issue. I'm not 100% sure, but it is something to recall when creating the production image.